integration_tests/dbt_project_evaluator/models/marts/intermediate/to_exclude/trino__int_model_11.sql

```diff
@@ -0,0 +1,3 @@
+-- depends on: {{ ref('fct_model_9') }}
+
+select 1 as id
```

Comment:
nit: add EOL

Reply:
it looks the same as `trino__int_model_10.sql`, can we get rid of one?

Reply:
These 2 models are located in `.../to_exclude/` path. They are used to test whether the [exclude_paths_from_project variable](https://github.com/starburstdata/dbt-trino-utils/blob/add-dbt-project-evaluator/integration_tests/dbt_project_evaluator/dbt_project.yml#L123) excludes all models from specified path. I would leave it as it is, to check if both models from this path are exluded (this is arranged in a same way as in [dbt-project-evalutor tests](https://github.com/dbt-labs/dbt-project-evaluator/tree/v0.6.2/integration_tests/models/marts/intermediate/to_exclude))